doc.html

```diff
@@ -0,0 +1,111 @@
+<html>
+<head>
+    <style type="text/css">
+        body { font-family: courier; font-size:12px; margin-left: 42px; width: 465px; white-space: pre-line; }
+        h2 { margin: 10px 0 0 -22px; font-size:12px; padding: 0 !important }
+        table { font-size: 10px; margin-left: -22px; width: 480px; overflow-wrap: break-work; }
+        table td { overflow-wrap: break-work; }
+        div.header { padding: 0; margin: 0 0 0 -22px; white-space: pre-wrap; }
+    </style>
+</head>
+<body>
+
+<div class="header">
+ATLAS Service Template Specification
+Name                                                    feed_data
+Version                                                     0.0.3
+Author(s)             ? (MmmooOgle), J. V&ouml;lker (fodjan GmbH)
+</div>
+
+<h2>The "feed_data" ATLAS Service Template</h2>
+<div style="margin-left: -22px;">
+
+<h2>Abstract</h2>
+This document describes the specification for the feed_data service whose purpose is to provide an interface for data-exchange of planned and actual feeding.
+
+</div>
+<h2>1 Introduction</h2>
+Exchange planned rations and actual feedingreports is an important step to keep animals healthy, control the feedcosts and reduce environmental impact by increasing the precision of feeding.
```

Comment:
In plain text, would it not be better to use "feeding reports" instead of feedingreports and "feed costs" instead of feedcosts?